src/Exercism.Analyzers.CSharp/Analyzers/Leap/LeapSolution.cs

```diff
@@ -7,17 +6,20 @@ namespace Exercism.Analyzers.CSharp.Analyzers.Leap
 {
     internal class LeapSolution : ParsedSolution
     {
-        public MethodDeclarationSyntax IsLeapYearMethod { get; }
-        public ParameterSyntax YearParameter { get; }
-        public ExpressionSyntax ReturnedExpression { get; }
+        public readonly ParameterSyntax YearParameter;
```

Comment:
Might I ask your reason for this change? I'm not opposed to it, but I'd like to hear your thoughts on it.

Reply:
I tried to make it the same as the gigasecond solution. But now I see two-fer uses the getters. So no real reason from my side.

Reply:
Ah. An inconsistency on my part then.